angular-rails-app/app/controllers/dob_permits_controller.rb

```diff
@@ -0,0 +1,12 @@
+class DobPermitsController < ApplicationController
+
+  def index 
+    if params[:property_id]
+      @dob_permits = DobPermit.where(
```

Comment:
Well, in theory, even with `group / order by` commands, the structure of the code would still remain the same. You'd just need to inject the particular symbols corresponding to the columns you wish to group or order by along with the model class (and maybe skip group or order if no symbols are provided).